### ISSUE TITLE
Next.js 12 complaining about "include" object property

### ISSUE THREAD

**Issue by DedaDev** [jeremybarbet/next-react-svg#13]
I'm getting `The root value has an unexpected property, include, which is not in the list of allowed properties...`

**How to reproduce**
   - Install next.js v12
   - add to your `next.config.js`
```js
    module.exports = withReactSvg({
      include: path.resolve(__dirname, 'src/assets'),
      webpack(config, options) {
        return config;
      },
    });
```

   - run dev server

**Comment by hasanm95** [comment]
I have a draft for this. I leave the link to the PR. You can review this.

[#14](https://github.com/jeremybarbet/next-react-svg/pull/14)

**Comment by jeremybarbet** [comment]
Released in 1.2.0, see README for config changes.
